### DISCUSSION TITLE
Is there a way to pass `--use-feature=truststore` to pip using Poetry?

### DISCUSSION THREAD

**Discussion by signed-log** [python-poetry/poetry#8729] [Q&A]
I have a project that communicates with hosts on a network which has centrally-signed HTTPS certificates.

These are in the Linux `ca-certificates` trusted store, but since Python uses hard-coded HTTPS cert bundle, I can't use those

There is a new [feature](https://pip.pypa.io/en/latest/topics/https-certificates/), `--use-feature=truststore` in `pip>=22.2` which would switch python from the hard-coded cert bundle to the system trust store

I can't find how to apply it through Poetry, is this possible?

**Comment by dimbleby** [comment]
poetry is not using pip.  If you want to use pip, just use pip

**Comment by Secrus** [comment]
I have it on my TODO list to add truststore support to Poetry, but it's not available right now

**Reply by abn** [reply]
Is there an issue number for this @Secrus ?

**Reply by Secrus** [reply]
Nope, just something I have planned regarding networking in Poetry, but that's only in my head for now.
